cass-operator.yaml

```diff
@@ -22,14 +22,28 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: f1f25eb8434876f4eb519a2e5504c171020916ad
 
-  - runs: go get golang.org/x/net@v0.17.0 # Mitigate CVE-2023-39325 and CVE-2023-3978
-
-  - runs: CGO_ENABLED=0 go build -a -o manager cmd/main.go
-
-  - runs: mkdir -p ${{targets.destdir}}/usr/bin; install -Dm755 ./manager ${{targets.destdir}}/usr/bin/manager
+  - uses: go/build
+    with:
+      modroot: .
+      packages: ./cmd
+      output: manager
+      # Mitigate CVE-2023-39325 and CVE-2023-3978
+      deps: golang.org/x/net@v0.17.0
 
   - uses: strip
 
+subpackages:
+  - name: k8ssandra-system-logger
```

Comment:
```suggestion
  - name: cass-operator-config
```

Reply:
This is a different package rather than a config actually, in `cass-operator` repo, there are 2 Dockerfiles.